app/services/youtube_service.rb

```diff
@@ -18,4 +32,20 @@ def conn
       f.params[:key] = ENV['YOUTUBE_API_KEY']
     end
   end
+
+  def add_to_total(new_results, total_results)
+    new_results.each do |result|
+      total_results << result unless result[:snippet][:title] == 'Private video'
+    end
```

Comment:
i feel like `[:thumbnails].nil?` and/or `[:thumbnails].empty?` would be a bit more robust. but this seems to be the next best thing.